uberfire-extensions/uberfire-runtime-plugins/uberfire-runtime-plugins-backend/src/main/java/org/uberfire/ext/plugin/backend/PluginServicesImpl.java

```diff
@@ -852,4 +870,21 @@ IOService getIoService() {
     private Path getMenuItemsPath(final Path rootPlugin) {
         return rootPlugin.resolve("info.dynamic");
     }
+
+    @Override
+    public org.uberfire.backend.vfs.Path save(final org.uberfire.backend.vfs.Path _path,
+                                              final Plugin content,
+                                              final DefaultMetadata _metadata,
+                                              final String comment) {
+        return save(content, comment);
```

Comment:
Similar question, can't be this implemented as below?
```java
saveAndRenameService.save(_path, content, _metadata, comment);
```

Reply:
I commented about it above ;-)